src/components/Dot.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import './Dot.css';
+
+const Dot = ({ id, active, dotClick }) => {
+  let name = active ? 'wui-dot wui-dot-active' : 'wui-dot';
+  return <div data-id={id} className={name} onClick={e => dotClick(parseInt(e.target.getAttribute('data-id')))} />;
```

Comment:
if you already have the `id` in function, then I think because of closure you would only need to do `onClick={e => dotClick(id)}`

Please also include proptypes

Reply:
Fixed.